code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -533,6 +540,9 @@
 /mob/living/simple_animal/slime/random/Initialize(mapload, new_colour, new_is_adult)
 	. = ..(mapload, pick(slime_colours), prob(50))
 
+/mob/living/simple_animal/slime/random_tier_one/Initialize(mapload, new_colour, new_is_adult=FALSE) ///A random slime that is blue, metal, orange, or purple.
```

Comment:
```suggestion
///A random slime that is blue, metal, orange, or purple.
/mob/living/simple_animal/slime/random_tier_one

/mob/living/simple_animal/slime/random_tier_one/Initialize(mapload, new_colour, new_is_adult=FALSE)
```

Reply:
This still isn't resolved
```suggestion
///A random slime that is blue, metal, orange, or purple.
/mob/living/simple_animal/slime/random_tier_one/Initialize(mapload, new_colour, new_is_adult=FALSE)
```

Reply:
Position of the comment has been updated